app/controllers/games_controller.rb

```diff
@@ -22,16 +22,39 @@ def create
   end
 
   def update
+    #this needs to be cleaned up seriously using private game_params
     @game = Game.find(params[:id])
-    if current_user.id == params[:jedi_user_id].to_i
+    if (current_user.id != params[:jedi_user_id].to_i) && (@game.sith_user_id == nil)
       @game.update(sith_user_id: current_user.id)
       redirect_to ships_path
-    elsif current_user.id == params[:sith_user_id].to_i
+    elsif current_user.id != params[:sith_user_id].to_i && (@game.jedi_user_id == nil)
       @game.update(jedi_user_id: current_user.id)
       redirect_to ships_path
     else
       render text: "Invalid Request", status: :unprocessable_entity
     end
   end
-  
+
+  private
+
+
+
```

Comment:
Very nice cleaning up this unused code.  Might make sense to remove it entirely, since it's no longer needed.  Thumbs up on the refactor though!